crates/agglayer-node/src/kernel/mod.rs

```diff
@@ -274,7 +273,7 @@ where
         Ok(call)
     }
 
-    /// Verify that the signer of the given [`SignedProof`] is the trusted
+    /// Verify that the signer of the given [`SignedTx`] is the trusted
```

Comment:
Seems to me the mentions of `SignedProof` are actually supposed to refer to `SignedTx`. Is this correct? I don't see anything called `SignedProof` in the code.

Reply:
I think that `SignedProof` was an old name